src/plugins/vis_default_editor/public/components/agg_params_helper.test.ts

```diff
@@ -190,7 +168,6 @@ describe('DefaultEditorAggParams helpers', () => {
         ],
         advanced: [],
       });
-      expect(agg.getIndexPattern).toBeCalledTimes(1);
```

Comment:
Wanted to call this out -- I removed this assertion because:
- it felt orthogonal to what's actually being tested here
- it failed when I moved agg type filters into the default editor, because one of the filters (rollup) calls `getIndexPattern` as well. that meant this assertion was failing because it is now called 2 times instead of 1.

if you'd prefer to keep the assertion, we'd need to change it to 2 instead.